### DISCUSSION TITLE
linphone

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1409] [Icon request] [answered]
ok so this VOIP app icon looks ugly when placed next to all of my macos icons lol

![linphone copy](https://user-images.githubusercontent.com/96893300/191173654-ed440c41-9705-487e-8ae0-d8fbb7f73931.jpg)

**Comment by rahman09abd** [comment, marked as answer]
I've uploaded (status: waiting) and you can check my profile later here [mmnk09](https://macosicons.com/#/u/mmnk09)

**Reply by ghost** [reply]
Thanks!
